### ISSUE TITLE
🏗️ Refactor home page oss.zenika.com

### ISSUE THREAD

**Issue by jeanphi-baconnais** [zenika-open-source/oss.zenika.com#396]
We currently have on the oss.zenika.com home page these information : 
- a banner
- highlighted projects (static elements in the project)
- statistics (from github-insight project)
- top projects (from github-insight project)
- open source white paper

❓ Do you think all information are pertinent and represents the Zenika open source DNA ?

You can write in comment what do you think about this page : positive or negative think, elements you want like to see (or not) in this page

**Comment by jeanphibaconnais** [comment]
In statistics area, some projects was created by people who left Zenika. They are on personal Github account so I think they could be removed from this page.

I will be curious to create database statistics to see how many contributions are made during all the year by all Zenika people and display them.

**Comment by gmembre-zenika** [comment]
The first visual part of the website feels "empty" : 
![image](https://user-images.githubusercontent.com/7294764/210534127-111be4b6-51a4-4b46-bbbb-79d3f3c2b382.png)

The number of contribution displayed lower should be move in this empty area

**Comment by bengeois** [comment]
👋 I agree with the refactor. We don't really highlight what we're doing/organizing.
The first times I went to the site, I felt a bit lost with all the information. We don't immediately understand that we're on the Zenika open-source site (I agree with @gmembre-zenika).

Having the statistics on the home page is a good idea, especially during hacktoberfest.
My vision is:
- A hero section with a gradient background, a headline and a countdown to the next hacktoberfest.
- At hacktoberfest, stats instead of countdown

Then the following sections:
- Upcoming events
- Global statistics (your idea of a database @jeanphibaconnais)
- Featured project
- Text section (presentation of open-source) or Video section (see videos on open-source)
- Footer

Then all other things (resources, etc.) on other pages accessible with the menu

**Comment by jeanphibaconnais** [comment]
Thanks for your feedback @bengeois 🙏 

For the Hacktoberfest part, we enabled the dashboard. Maybe we can enable the Hacktoberfest part (and this countdown) at the beginning of september.

What do you mean when you say `Featured project` ? Do you have any examples ?

**Comment by bengeois** [comment]
Yes, enabling the countdown beginning of September can be a good idea. And for all others months, I think we should use this first section to highlight other things but not having a picture only.

By `Featured project`, I mean "active" or "newest" project in the Zenika Open Source organisation (like cheatsheet) or projects where Z are active maintainer 👍🏼 

This is what I think pertinent on the homepage

**Comment by jeanphi-baconnais** [comment]
Closed with https://github.com/zenika-open-source/oss.zenika.com/pull/492
